tox.ini

```diff
@@ -1,20 +1,17 @@
 [tox]
-envlist = py27-{sqlite,pg}
+envlist = py27-{django16,django17,django18}
```

Comment:
Requires Tox>=1.8.

It can be defined as `tests_require` or `extras_require` in `setup.py`.
Something like `setup(extras_require={'test': ['tox>=1.8']})`. Then you can `pip install -e .[test]` in Makefile or travis.ini.
See https://github.com/novafloss/pydocusign/blob/master/setup.py for ideas.